test/cppsuite/test_harness/workload/random_generator.cxx

```diff
@@ -37,25 +42,24 @@ random_generator::instance()
 }
 
 std::string
-random_generator::generate_string(std::size_t length)
+random_generator::generate_string(std::size_t length, characters_type type)
 {
-    std::string random_string;
-
-    for (std::size_t i = 0; i < length; ++i)
-        random_string += _characters[_distribution(_generator)];
-
-    return (random_string);
+    std::string str(get_characters(type));
+    std::shuffle(str.begin(), str.end(), _generator);
+    return (str.substr(0, length));
```

Comment:
What if `length` is greater than the number of characters in `str`?

Reply:
Good finding! See 33103615a7cf676c874769e60f977fb34e548cc0